jupiterone/client.py

```diff
@@ -105,12 +105,19 @@ def _execute_query(self, query: str, variables: Dict = None) -> Dict:
                     raise JupiterOneApiError(content.get('errors'))
                 return response.json()
 
+        elif response.status_code == 401:
+            raise JupiterOneApiError('401: Unauthorized. Please supply a valid token')
+
         elif response.status_code in [429, 500]:
             raise JupiterOneApiRetryError('JupiterOne API rate limit exceeded')
 
         else:
-            content = json.loads(response._content)
-            raise JupiterOneApiError('{}:{}'.format(response.status_code, content.get('error')))
+            try:
+                content = json.loads(response._content)
+                raise JupiterOneApiError('{}: {}'.format(response.status_code, content.get('error') or 'Unknown Error'))
```

Comment:
I'm a bit rusty with Python, but I think the `or` operator doesn't handle nullish coalescing like we'd expect in a TypeScript world.  I'm assuming that's why we need the additional try/catch here to catch the ValueError?

We could do some kind of if statement optionally, but I think this solution is fine too.

Reply:
Nice! So is the additional try/catch to handle something else, or more of a belt and suspenders kind of thing?